andrea/js/localStorage.js

```diff
@@ -0,0 +1,7 @@
+module.exports =  new class {
+  store = {};
+  setItem = (key, val) => (this.store[key] = val);
+  getItem = key => this.store[key];
+  removeItem = key => { delete this.store[key]; };
```

Comment:
removeItem = key => { if(this.store.hasOwnProperty(key)){delete this.store[key]; return true;} return false;};